src/osd/ReplicatedBackend.h

```diff
@@ -114,15 +114,15 @@ class ReplicatedBackend : public PGBackend {
     }
     {
       f->open_array_section("pushing");
-      for (std::map<hobject_t, std::map<pg_shard_t, PushInfo>>::const_iterator i =
+      for (std::map<hobject_t, std::map<pg_shard_t, push_info_t>>::const_iterator i =
```

Comment:
auto, and using cbegin()/cend()?

Reply:
How about switching to the range-based `for`?